docs/btc-methods.md

```diff
@@ -0,0 +1,73 @@
+# BTC Methods
+
+Here we document the Bitcoin methods that an account Snap may implement to
+support requests originated from dapps.
+
+## btc_sendmany
+
+This method is similar to the `sendmany` RPC method from Bitcoin Core, but its
+parameters are passed in an object instead of an array, and are named in
+camelCase. Also, dummy values aren't allowed.
+
+### Parameters
+
+- **Request object (required):**
```

Comment:
We are naming those differently with our EVM methods, like: `Transaction (required)`, but since we are adapting an existing API, maybe `Request object` works fine here.

Otherwise, `Transaction intent` or `Transaction` could eventually work here?

Reply:
Indeed, `Transaction intent` looks better.